### PR TITLE
Fix several typos

### PR DESCRIPTION
**Pull request by JonHanna**
coment → comment
rangecheck → range check
testcase → test case
deffer → defer
everytime → every time
whitespaces → whitespace (whitespace is uncountable)
white space → whitespace
white spaces → whitespace
the the → the
a a → a
nametest → name test
scaned → scanned
and and → and
undecision → indecision
are no whitespace → is no whitespace
Whidbey casing
Everett casing
unkown → unknown
messsage → message
mesage → message
mesasage → message
publically → publicly (publically is attested but questionable).
exceptionto → exception to
mutatable → mutable
implemention → implementation
hadnt → hadn't
couldnt → couldn't
wouldnt → wouldn't
wont → won't (slightly more correct *wont* than I would have expected, mostly Shakespeare in test data 😄 )
isntead → instead
preocessor → processor
particural → particular
interations → iterations
enterying → entering
stoppped → stopped
stoped → stopped
verication → verification
compuatation → computation
revisisted → revisited
partioner → partitioner
lock'ed → locked
table'd → table would
hashtable'ed → hashtabled (still coining, but tabled is analogous)
intialize → initialize
calcuate → calculate
perfom → perform
performr → perform
matthing → matching
corect → correct
thorw → throw
supress → suppress
reqeust → request
mutliple → multiple
ot → to
trival → trivial
upto → up to
specfices → specifies
patttern → pattern
descrepancy → discrepancy
explict → explicit
implict → implicit
usign → using
non-sensical → nonsensical
contination → continuation
producs → products
childred → children
parametr → parameter
approriate → appropriate
insted → instead
expection → exception
successsfully → successfully
funtion → function
unqiue → unique
apparant → apparent
expception → exception
conatined → contained
apssed → passed
propogate → propagate
cutoms → custom
currrent → current
curent → current
concurent → concurrent
coplted → completed
unexcepted → Unexpected (where the latter seems more appropriate)
to to → to
abd → and
refence → reference
particiant → participant
cuurrent → current
extratc → extract
cparticipant → participant
ehich → which
choosen → chosen
mske → make
alread → already
aquire → acquire
droppping → dropping
intial → initial
tru → true
aprticipant → participant
excption → exception
sempahore → semaphore
fileds → fields
threashold → threshold
duting → during
didnt → didn't
atribute → attribute
serliazation → serialization
localy → locally
channge → change
prperties → properties
attrbute → attribute
sentinal → sentinel
shoukd → should
existant → existent
commnent → comment
expeted → expected
propert → property
commnet → comment
datat → data
ther → the
mach → match
inhertiance → inheritance
convience → convenience
completley → completely
otheriwse → otherwise
actuall → actual
inefficent → inefficient
defalut → default
comparsion → comparison
charcters → characters
postion → position
aleady → already
charater → character
incosistent → inconsistent
seperate → separate
veify → verify
currentlly → currently
corect → correct
betwwen → between
omiting → omitting
readr → reader
compae → compare
allways → always
thown → thrown
with with → with
usecase → use case
avare → aware
lexem → lexeme
grammer → grammar
oposite → opposite
overrided → overridden
everithig → everything
chages → changes
memeber → member
proerties → properties
identifer → identifier
doesnot → does not
deriaved → derived
varialble → variable

### CODE REVIEW

File: src/Common/tests/System/Xml/ModuleCore/cmodinfo.cs [hunk 1]
```diff
@@ -71,7 +71,7 @@ public static string CommandLine
         {
             get
             {
-                //Deffered Parsing
+                //Defered Parsing
```
Comment (stephentoub) [inline src/Common/tests/System/Xml/ModuleCore/cmodinfo.cs]: Deferred has two r's.
(When you address PR feedback, please do it in a separate commit so I don't need to re-review 340 files again ;)

File: src/System.ComponentModel.TypeConverter/tests/PropertyDescriptorCollectionTests.cs [hunk 2]
```diff
@@ -168,7 +168,7 @@ public void RemoveAtTests(int index)
         [InlineData("propertyDescriptor2", true, true)]
         [InlineData("propertyDescriptor8", true, true)]
         [InlineData("propertyDescriptor9", true, true)]
-        [InlineData("propertyDescriptorNotExistant", true, false)]
+        [InlineData("propertyDescriptorNotexistent", true, false)]
```
Comment (stephentoub) [inline src/System.ComponentModel.TypeConverter/tests/PropertyDescriptorCollectionTests.cs]: Doesn't really matter, but why did you lower case the E?

File: src/Native/Windows/clrcompression/zlib-intel/deflate.c [hunk 3]
```diff
@@ -182,7 +182,7 @@ struct static_tree_desc_s {int dummy;}; /* for buggy compilers */
  * the previous length of the hash chain.
```
Comment (stephentoub) [inline src/Native/Windows/clrcompression/zlib-intel/deflate.c]: Please revert all changes to all files under clrcompression.

File: src/System.ComponentModel.TypeConverter/tests/PropertyDescriptorCollectionTests.cs (hunk 2 above)
Comment (stephentoub) [inline src/System.ComponentModel.TypeConverter/tests/PropertyDescriptorCollectionTests.cs]: Same lowercase E question

File: src/System.Data.Odbc/src/System/Data/Odbc/OdbcParameter.cs [hunk 4]
```diff
@@ -346,7 +346,7 @@ private int GetColumnSize(object value, int offset, int ordinal)
             {
                 if (ODBC32.SQL_C.NUMERIC == _typemap._sql_c)
                 {
-                    cch = 62;  // (DecimalMaxPrecision+sign+terminator)*BytesPerUnicodeCharater
+                    cch = 62;  // (DecimalMaxPrecision+sign+terminator)*BytesPerUnicodeCharacter
```
Comment (stephentoub) [inline src/System.Data.Odbc/src/System/Data/Odbc/OdbcParameter.cs]: Where is BytesPerUnicodeCharacter defined, and how does this compile if you only change the usage site and not the declaration?

File: src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/DiagnosticSourceEventSource.cs [hunk 5]
```diff
@@ -96,7 +96,7 @@ namespace System.Diagnostics
     /// 
     /// * How data is logged in the EventSource 
     /// 
-    /// By default all data from Diagnostic sources is logged to the the DiagnosticEventSouce event called 'Event' 
+    /// By default all data from Diagnostic sources is logged to the DiagnosticEventSouce event called 'Event'
```
Comment (stephentoub) [inline src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/DiagnosticSourceEventSource.cs]: As long as you're changing this line, it should be either "diagnostics sources" or "DiagnosticSources", but not "Diagnostic sources".

File: src/System.Private.Xml.Linq/tests/xNodeBuilder/XmlReaderDiff.cs [hunk 6]
```diff
@@ -126,7 +126,7 @@ public static void CompareNamespaceAware(XmlReader originalReader, XmlReader fil
                         }
                     }
 
-                    // does not underreach the index
+                    // does not under-reach the index
```
Comment (stephentoub) [inline src/System.Private.Xml.Linq/tests/xNodeBuilder/XmlReaderDiff.cs]: Why did you add a dash here?

File: src/System.ComponentModel.TypeConverter/tests/PropertyDescriptorCollectionTests.cs (hunk 2 above)
Comment (JonHanna) [inline src/System.ComponentModel.TypeConverter/tests/PropertyDescriptorCollectionTests.cs]: A typo in my fix of the typo.

File: src/System.Private.Xml/src/System/Xml/Serialization/SoapReflectionImporter.cs [hunk 7]
```diff
@@ -392,15 +392,15 @@ private bool InitializeStructMembers(StructMapping mapping, StructModel model, R
             {
                 StructMapping baseMapping = ImportStructLikeMapping((StructModel)_modelScope.GetTypeModel(model.Type.GetTypeInfo().BaseType, false), limiter);
 
-                // check to see if the import of the baseMapping was deffered
+                // check to see if the import of the baseMapping was defered
```
Comment (stephentoub) [inline src/System.Private.Xml/src/System/Xml/Serialization/SoapReflectionImporter.cs]: Two r's in deferred, for all of these

File: src/System.Security.Cryptography.X509Certificates/tests/X509StoreTests.cs [hunk 8]
```diff
@@ -113,7 +113,7 @@ public static void ReadMyCertificates()
         }
 
         [Fact]
-        public static void OpenNotExistant()
+        public static void OpenNotexistent()
```
Comment (stephentoub) [inline src/System.Security.Cryptography.X509Certificates/tests/X509StoreTests.cs]: Another unexpected capital to lower-case E

File: src/System.Private.Xml.Linq/tests/xNodeBuilder/XmlReaderDiff.cs (hunk 6 above)
Comment (JonHanna) [inline src/System.Private.Xml.Linq/tests/xNodeBuilder/XmlReaderDiff.cs]: Not a closed compound I was familiar with, but google tells me I was wrong.

File: src/System.Data.Odbc/src/System/Data/Odbc/OdbcParameter.cs (hunk 4 above)
Comment (JonHanna) [inline src/System.Data.Odbc/src/System/Data/Odbc/OdbcParameter.cs]: It's a comment in C# style explaining how the constant was arrived at, rather than a real usage site.

File: src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/DiagnosticSourceEventSource.cs (hunk 5 above)
Comment (JonHanna) [inline src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/DiagnosticSourceEventSource.cs]: The latter, with the type name being used, seems most reasonable and in keeping with the rest.

**Comment by JonHanna** [comment]
Each addressed in separate small commits, though I'm not sure if github's auto-squash will leave clrcompression truly untouched.

**Comment by stephentoub** [comment]
Thanks, @JonHanna.
